sync-core/src/main/java/com/cloudant/sync/query/QueryValidator.java

```diff
@@ -26,10 +26,19 @@
  */
 class QueryValidator {
 
-    public static final String AND = "$and";
-    public static final String OR = "$or";
-    public static final String EQ = "$eq";
-
+    private static final String AND = "$and";
```

Comment:
These strings also appear in QuerySqlTranslator.java and UnindexedMatcher.java - can we have them defined in one place?